applications_tests/lethe-fluid/mortar_taylorcouette.output

```diff
@@ -0,0 +1,34 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       128
+   Number of degrees of freedom: 1312
+   Volume of triangulation:      2.9451
+
+*****************************
+Steady iteration:        1/4
+*****************************
+
+*****************************
+Steady iteration:        2/4
+*****************************
+   Number of active cells:       512
+   Number of degrees of freedom: 4928
+   Volume of triangulation:      2.94523
+
+*****************************
+Steady iteration:        3/4
+*****************************
+   Number of active cells:       2048
+   Number of degrees of freedom: 19072
+   Volume of triangulation:      2.94524
+
+*****************************
+Steady iteration:        4/4
+*****************************
+   Number of active cells:       8192
+   Number of degrees of freedom: 75008
+   Volume of triangulation:      2.94524
```

Comment:
I would remove this iteration, on the debug CI a 10k cell Q2-Q2 is going to require a lot of time, anyway you already have the converged order before so I don't think it is necessary